TROUBLESHOOTING.md

```diff
@@ -5,7 +5,13 @@
 * [How to debug log files](#debug-logs)
 
 ### Common Issues
+* [Databricks Cluster Won't Start](#cluster-fails)
 * [I don't see lineage in Microsoft Purview](#no-lineage)
+* [Databricks Cluster Won't Start](#cluster-fails)
+* [Databricks Driver Logs: EventEmitter Could Not Emit Lineage](#driver-log-eventemitter)
+* [Error Loading to Purview: 403 Forbidden](#pureviewout-load2purview)
+* [Missing Ol to Purview mapping data for this source](#purviewout-olmapping)
```

Comment:
Update all instances of "Ol" to either OL or OpenLineage

Reply:
In this case, Ol is part of the error message being emitted from the PurviewOut logs.